datasets/rureviews/rureviews.py

```diff
@@ -0,0 +1,94 @@
+
+Since the data were collected from the top-ranked goods with high average rating, the amount of positive reviews considerably exceeded the amount of neutral and negative reviews. At the same time, we have no information about real-life class distribution for reviews, that is why we decided to use an undersampling technique to deal with imbalanced data. For each class 30k of reviews were randomly selected to make balanced dataset.
+"""
+
+_CITATION = """\
+@INPROCEEDINGS{Smetanin-SA-2019,
+  author={Sergey Smetanin and Michail Komarov},
+  booktitle={2019 IEEE 21st Conference on Business Informatics (CBI)},
+  title={Sentiment Analysis of Product Reviews in Russian using Convolutional Neural Networks},
+  year={2019},
+  volume={01},
+  number={},
+  pages={482-486},
+  doi={10.1109/CBI.2019.00062},
+  ISSN={2378-1963},
+  month={July}
+}
+"""
+
+
+_TRAIN_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/sismetanin/rureviews/master/women-clothing-accessories.3-class.balanced.csv"
+)
+
+
+class AppReviews(datasets.GeneratorBasedBuilder):
+    """RuReviews: An Automatically Annotated Sentiment Analysis Dataset for Product Reviews in Russian"""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "review": datasets.Value("string"),
+                    "sentiment": datasets.Value("string"),
+                }
+            ),
+            homepage="https://ieeexplore.ieee.org/document/8807792",
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        train_path = dl_manager.download_and_extract(_TRAIN_DOWNLOAD_URL)
+        return [
+            datasets.SplitGenerator(name=datasets.Split.TRAIN, gen_kwargs={"filepath": train_path}),
+        ]
+
+    def _generate_examples(self, filepath):
+        """Generate Distaster Response Messages examples."""
+        with open(filepath, encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(
+                csv_file, quotechar='"', delimiter=",", quoting=csv.QUOTE_ALL, skipinitialspace=True
+            )
+            next(csv_reader, None)
+
+            for id_, row in enumerate(csv_reader):
+                row = [s for s in row if "\t" in s]
+                row = row[0].split("\t")
+                if len(row) == 2:
+                    review, sentiment = row
+                elif len(row) == 3 or len(row) == 4:
+                    review = " ".join(row[:-1])
+                    sentiment = row[-1]
```

Comment:
In the dataset csv there's a typo for the `neutral` sentiment. Can you fix it please ?
```suggestion
                    sentiment = row[-1]
                    if sentiment == "neautral":
                        sentiment = "neutral"
```